tests/test_main.py

```diff
@@ -700,6 +700,22 @@ def test_link(request_ctx):
         assert b"<!DOCTYPE html>" in rv.data, "Expected HTML content"
 
 
+def test_faq(request_ctx):
```

Comment:
1) better to user for that client fixture;
2) it can be done in one test with parameters, eg,
```
@pytest.mark.parametrize("url", ["/faq", "/about"])
def test_queries(client, url):
   ....
```

Reply:
rv = client.get(url)